app/uk/gov/hmrc/mobilestatus/config/AppConfig.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.mobilestatus.config
+
+import play.api.Configuration
+
+import java.nio.charset.StandardCharsets
+import javax.inject.{Inject, Singleton}
+
+@Singleton
+class AppConfig @Inject() (config: Configuration) {
+
+  def nameOfConfigFile: Option[String] =
+    config
+      .getOptional[String]("nameOfConfigFile")
```

Comment:
Is this right? Does this infer that the file name is `nameOfConfigFile`?

Reply:
It's referencing a config property that's value is the name of the file. So we can use different files for testing.